FirstProject/assets/grid.less

```diff
@@ -0,0 +1,39 @@
+.grid {
+  width: 1200px;
+  display: flex;
+  margin: 0 auto;
+}
+
+.col-1 {
+  flex: 0 0 33.33%;
+}
+
+.col-2 {
+  flex: 0 0 68.68%;
+}
+
+.col-3 {
+  flex: 0 0 100%
+}
+
+.col-offset-2 {
+
+}
+
+@media (min-width: 768px) and (max-width: 1200px) {
```

Comment:
If this media query isn't going to be augmented to include other rules as well, you might want to consider rewriting it to look a bit like this:
```css
@media (max-width: 1200px) {
    .grid {
        max-width: 100%;
    }
}
```
If you do this, then below you can clean up the `.grid` for `max-width: 768px;` to remove the `max-width: 100%` rule, since it will already be inherited from the `max-width: 1200px` rule. In fact, this is a common pattern of desktop-first responsive design.

For the alternative, mobile-first responsive design, you would start off with the styling applicable on your supported mobile devices from the lowest supported screen size and work your way up to desktop in increasing order of the screen dimensions, overriding just the parts you need, starting with your desired breakpoint. That way, you only have to specify common styling once, keeping your code nice and DRY (btw, are you guys familiar with the DRY principle?).